dolphinscheduler-tools/src/main/java/org/apache/dolphinscheduler/tools/command/CommandApplication.java

```diff
@@ -18,135 +18,27 @@
-                        "    `last_update_time` timestamp    NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP COMMENT 'last update time',\n"
-                        +
-                        "    `create_time`      timestamp    NOT NULL DEFAULT CURRENT_TIMESTAMP COMMENT 'create time',\n"
-                        +
-                        "    PRIMARY KEY (`id`),\n" +
-                        "    unique (`lock_key`)\n" +
-                        ") ENGINE = InnoDB\n" +
-                        "  DEFAULT CHARSET = utf8;");
-            }
-        }
-
-        private void jdbcRegistrySchemaInitializeInPG() throws SQLException {
-            try (
-                    Connection connection = dataSource.getConnection();
-                    Statement statement = connection.createStatement()) {
-                statement.execute("create table t_ds_jdbc_registry_data\n" +
-                        "(\n" +
-                        "    id               serial\n" +
-                        "        constraint t_ds_jdbc_registry_data_pk primary key,\n" +
-                        "    data_key         varchar                             not null,\n" +
-                        "    data_value       text                                not null,\n" +
-                        "    data_type        int4                                not null,\n" +
-                        "    last_term        bigint                              not null,\n" +
-                        "    last_update_time timestamp default current_timestamp not null,\n" +
-                        "    create_time      timestamp default current_timestamp not null\n" +
-                        ");");
-                statement.execute(
-                        "create unique index t_ds_jdbc_registry_data_key_uindex on t_ds_jdbc_registry_data (data_key);");
-                statement.execute("create table t_ds_jdbc_registry_lock\n" +
-                        "(\n" +
-                        "    id               serial\n" +
-                        "        constraint t_ds_jdbc_registry_lock_pk primary key,\n" +
-                        "    lock_key         varchar                             not null,\n" +
-                        "    lock_owner       varchar                             not null,\n" +
-                        "    last_term        bigint                              not null,\n" +
-                        "    last_update_time timestamp default current_timestamp not null,\n" +
-                        "    create_time      timestamp default current_timestamp not null\n" +
-                        ");");
-                statement.execute(
-                        "create unique index t_ds_jdbc_registry_lock_key_uindex on t_ds_jdbc_registry_lock (lock_key);");
-            }
-        }
-
+    @Override
+    public void run(String... args) throws Exception {
+        jdbcRegistrySchemaInitializeCommand.run(args);
```

Comment:
We should add this to the upgrade docs.

Reply:
I added the jdbc registry table into the initialize/upgrade sql, this means users can directly use jdbc as registry without initialize the table again